docs/tasks/restore_file.md

```diff
@@ -130,9 +133,12 @@ The following information is copied directly from the included task in this role
     wait_for_job: "{{ cohesity_restore_file.wait_for_job | default('yes') }}"
     wait_minutes: "{{ cohesity_restore_file.wait_minutes | default(10) }}"
     overwrite: "{{ cohesity_restore_file.overwrite | default('yes') }}"
+    vm_name: "{{ cohesity_restore_file.vm_name | default('') }}"
     preserve_attributes: "{{ cohesity_restore_file.preserve_attributes | default('yes') }}"
     restore_location: "{{ cohesity_restore_file.restore_location | default('') }}"
-    backup_timestamp: "{{cohesity_restore_file.backup_timestamp | default('') }}"
+    backup_timestamp: "{{ cohesity_restore_file.backup_timestamp | default('') }}"
+    vm_password: "{{ cohesity_restore_file.vm_password | default('') }}"
```

Comment:
How would one distinguish vmware file restore from physical file restore, should these be 2 separate modules ?

Reply:
Added new files for vmware.